src/docker/build.nim

```diff
@@ -269,10 +269,25 @@ proc launchDockerSubscan(ctx:     DockerInvocation,
   result = runChalkSubScan(usableContexts, "insert").report
   trace("docker: subscan complete.")
 
+proc getRelativePathBetween(from_path: string, to_path: string) : string =
+  ## Given the `from_path`, usually the project root, return the relative
+  ## path of the file's `to_path`. Return nothing if its outside the project root
+  ## or if `to_path` is an empty string.
+  let relativePath = to_path.relativePath(from_path)
```

Comment:
no need to change. just fyi this can be rewritten in nim as:

```nim
result = "foo"
if <condition>:
  return ""
```

`result` will be returned by default within the function but it still allows to access it if needed for example for logging/etc